pymongo/operations.py

```diff
@@ -24,7 +35,7 @@
 
 # Hint supports index name, "myIndex", or list of either strings or index pairs: [('x', 1), ('y', -1), 'z'']
 _IndexList = Sequence[Union[str, Tuple[str, Union[int, str, Mapping[str, Any]]]]]
-_IndexKeyHint = Union[str, _IndexList]
+_IndexKeyHint = Union[str, _IndexList, ItemsView[str, Any]]
```

Comment:
Is this needed? Shouldn't ItemsView should already be supported by _IndexList because it's a sequence of tuples?

Reply:
No, mypy treated them differently.  The test was giving the error:

```python
test/test_collection.py: note: In member "test_create_index" of class "TestCollection":
test/test_collection.py:320: error: Argument 1 to "create_index" of "Collection" has incompatible type
"dict_items[str, int]"; expected "Union[str, Sequence[Union[str, Tuple[str, Union[int, str, Mapping[str, Any]]]]]]"
[arg-type]
            db.test.create_index({'hello': 1}.items())
```

Reply:
Let's not change our type signature here. "supporting" ItemsView was a mistake, let's not go more out of our way to support it. This looks like a glaring mypy bug to me.

Can we add a type ignore to the test instead of further complicating the type signature?

Reply:
Done

Reply:
Thanks!